intranet/static/js/common.header.js

```diff
@@ -57,7 +63,7 @@ $(function() {
             $(document).unbind("click tap", clickOutside);
             $(".dropdown-menu").unbind("click", clickInside);
             $(".has-dropdown").removeClass("dropdown-open");
-        } else if (already_open.length == 0) {
+        } else if (already_open.length === 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'already_open' is not in camel case.](https://www.codacy.com/app/tjcsl/ion/file/2769577516/issues/source?bid=3176160&fileBranchId=3355598#l66)